app/views/moderations/actions_panel.html.erb

```diff
@@ -154,22 +154,55 @@
       </button>
       <% end %>
       <a href="<%= URL.url %>/community-moderation#how-to-rate-the-experience-level-of-a-post" target="_blank" rel="noopener">
-        <span class="how-this-works-section">
+        <span class="additional-subtext-section py-3">
           <%= t("views.moderations.actions.how") %>
         </span>
       </a>
     </div>
 
     <% if current_user.any_admin? && @moderatable.published %>
-      <div class="unpublish-article">
-        <button
-          class="crayons-btn crayons-btn--danger"
-          id="unpublish-article-btn"
-          data-article-id="<%= @moderatable.id %>"
-          data-article-author="<%= @moderatable.username %>"
-          data-article-slug="<%= @moderatable.slug %>">
-          <%= t("views.moderations.actions.unpublish") %>
-        </button>
+      <button aria-haspopup="true" aria-expanded="false" aria-controls="admin-action-options" class="other-things-btn admin-action" type="button" data-other-things-type="admin-action" aria-label="<%= t("views.moderations.actions.admin.aria_label") %>">
+        <div class="label-wrapper">
+          <div class="icon circle centered-icon">
+            <%= inline_svg_tag("admin.svg", aria: true, title: t("views.moderations.actions.experience.icon")) %>
+          </div>
+          <header>
+            <h2><%= t("views.moderations.actions.admin.heading") %></h2>
+            <h3><%= t("views.moderations.actions.admin.subtitle") %></h3>
+          </header>
```

Comment:
We shouldn't really have headers and headings inside button elements, but I can see this is the pattern we have for the other buttons, so better left for a later refactor